src/tools/s2wasm.cpp

```diff
@@ -33,6 +34,7 @@ int main(int argc, const char *argv[]) {
   bool generateEmscriptenGlue = false;
   std::string startFunction;
   std::vector<std::string> archiveLibraries;
+  bool validate = false;
```

Comment:
OK, except not all the output (e.g. when you run the binaryen tests) passes validation yet. And I'd like to have the ability to run it separately from fixing the invalid cases. So can we land this for now and then make it mandatory once the errors are fixed?